tests/data/FileFactory.ts

```diff
@@ -0,0 +1,24 @@
+import { File } from '../../types';
+
+const ONE_BYTE_FILLER = 'a';
+
+// Factory for mocking in-memory files for testing with multer
+export class FileFactory {
```

Comment:
Do we still need this or can we mock the `File`?

Reply:
I think we'd still need this for verifying that a file that's too large gets rejected by the controller (the TODO test).